interfaces/builtin/tmpfs_mount_test.go

```diff
@@ -0,0 +1,100 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/builtin"
+	"github.com/snapcore/snapd/interfaces/seccomp"
+	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type TmpfsMountInterfaceSuite struct {
+	iface    interfaces.Interface
+	slotInfo *snap.SlotInfo
+	slot     *interfaces.ConnectedSlot
+	plugInfo *snap.PlugInfo
+	plug     *interfaces.ConnectedPlug
+}
+
+const tmpfsMountConsumerYaml = `name: consumer
+version: 0
+apps:
+ app:
+  plugs: [tmpfs-mount]
+`
+
+const tmpfsMountCoreYaml = `name: core
+version: 0
+type: os
+slots:
+  tmpfs-mount:
+`
+
+var _ = Suite(&TmpfsMountInterfaceSuite{
+	iface: builtin.MustInterface("tmpfs-mount"),
+})
+
+func (s *TmpfsMountInterfaceSuite) SetUpTest(c *C) {
+	s.plug, s.plugInfo = MockConnectedPlug(c, tmpfsMountConsumerYaml, nil, "tmpfs-mount")
+	s.slot, s.slotInfo = MockConnectedSlot(c, tmpfsMountCoreYaml, nil, "tmpfs-mount")
+}
+
+func (s *TmpfsMountInterfaceSuite) TestName(c *C) {
+	c.Assert(s.iface.Name(), Equals, "tmpfs-mount")
+}
+
+func (s *TmpfsMountInterfaceSuite) TestSanitizeSlot(c *C) {
+	c.Assert(interfaces.BeforePrepareSlot(s.iface, s.slotInfo), IsNil)
+}
+
+func (s *TmpfsMountInterfaceSuite) TestSanitizePlug(c *C) {
+	c.Assert(interfaces.BeforePreparePlug(s.iface, s.plugInfo), IsNil)
+}
+
+func (s *TmpfsMountInterfaceSuite) TestAppArmorSpec(c *C) {
+	spec := &apparmor.Specification{}
+	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.slot), IsNil)
+	c.Assert(spec.SecurityTags(), DeepEquals, []string{"snap.consumer.app"})
+	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, `#deny /run/mount/utab w,`)
```

Comment:
there are other interfaces which have this line too, can you change this check to something more uniquely from this interface such as `
mount fstype=tmpfs {none,tmpfs} -&gt; /var/snap/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/@{SNAP_REVISION}/{,**},` ?